app/build.gradle

```diff
@@ -36,13 +26,13 @@ shadowJar {
     }
 
     dependencies {
-        exclude(dependency("org.openjfx:javafx-base:11:$platform"))
-        exclude(dependency("org.openjfx:javafx-controls:11:$platform"))
-        exclude(dependency("org.openjfx:javafx-fxml:11:$platform"))
-        exclude(dependency("org.openjfx:javafx-graphics:11:$platform"))
-        exclude(dependency("org.openjfx:javafx-media:11:$platform"))
-        exclude(dependency("org.openjfx:javafx-swing:11:$platform"))
-        exclude(dependency("org.openjfx:javafx-web:11:$platform"))
+        exclude(dependency("org.openjfx:javafx-base:$javafx.version:$javafx.platform"))

```

Comment:
`$javafx.platform` should be changed to `$javafx.platform.classifier` for all the exclude directives.